OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -715,4 +716,7 @@ cd "$(SolutionDir)"</PostBuildEvent>
   <Target Name="AfterBuild">
   </Target>
   -->
+  <ItemGroup>
```

Comment:
I think our policy is to not merge these groups (which aren't actually necessary).